invenio_vocabularies/contrib/names/config.py

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
```suggestion
```
I think there's an extra line at the beginning of the file (don't know if it matters thought for the `# -*- coding: utf-8 -*-` to be detected.